gherkin/javascript/src/pickles/compile.ts

```diff
@@ -79,19 +79,13 @@ function compileRule(
   rule.children.forEach((stepsContainer) => {
     if (stepsContainer.background) {
       ruleBackgroundSteps = ruleBackgroundSteps.concat(stepsContainer.background.steps)
-    } else if (stepsContainer.scenario.examples.length === 0) {
-      compileScenario(
-        tags,
-        ruleBackgroundSteps,
-        stepsContainer.scenario,
-        language,
-        pickles,
-        uri,
-        newId
-      )
     } else {
-      compileScenarioOutline(
-        featureTags,
+      const compileFunc = stepsContainer.scenario.examples.length === 0 
```

Comment:
While not as short, could you use the same pattern for this as in Ruby? Having the same structure makes it easier to compare across language's.

Reply:
Sure, but I am on my vacation without a laptop. I will do this asap！ Thanks！

Reply:
Sorry for the delay of such a long time. I just committed a patch.